parts/common/components.json

```diff
@@ -1467,15 +1467,6 @@
                 "latestVersion": "1.34.0-ubuntu22.04u1"
               }
             ]
-          },
-          "r2004": {
-            "versionsV2": [
-              {
-                "k8sVersion": "1.34",
-                "renovateTag": "name=azure-acr-credential-provider, repository=production, os=ubuntu, release=20.04",
-                "latestVersion": "1.34.0-ubuntu20.04u1"
```

Comment:
why not stick to 1.33 ?

Reply:
we already cache 1.33.2 from MCR

Reply:
will let @lilypan26 weigh in before merging